src/MVC.Courses.Test.Integration/IisServerTestBase.cs

```diff
@@ -46,16 +45,16 @@ private void StartIIS()
             var programFiles = Environment.GetFolderPath(Environment.SpecialFolder.ProgramFiles);
 
             _iisProcess = new Process();
-            _iisProcess.StartInfo.FileName = programFiles + @"\IIS Express\iisexpress.exe";
-            _iisProcess.StartInfo.Arguments = string.Format("/path:\"{0}\" /port:{1}", applicationPath, IisPort);
+            _iisProcess.StartInfo.FileName = Path.Combine(programFiles,"IIS Express","iisexpress.exe");
+            _iisProcess.StartInfo.Arguments = $"/path:\"{applicationPath}\" /port:{IisPort}";
             _iisProcess.Start();
         }
 
         protected virtual string GetApplicationPath(string applicationName)
         {
-            var solutionFolder =
-                Path.GetDirectoryName(
-                    Path.GetDirectoryName(Path.GetDirectoryName(AppDomain.CurrentDomain.BaseDirectory)));
+            var baseDir = AppDomain.CurrentDomain.BaseDirectory;            
+            var solutionFolder = Path.GetFullPath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "..", "..",".."));
```

Comment:
this makes sense - that said, I am planning on getting IIS Express started directly from the build script. We will loose the ability to run the tests in the IDE - but this will make the solution more robust - especially once we add DB dependencies